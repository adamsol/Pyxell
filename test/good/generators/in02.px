
func f(): Int... def
    yield 1
    print 0

g = f()
print 1 in g
print 1 in g
print 1 not in g

g = f()
print 0 in g
print 1 in g
