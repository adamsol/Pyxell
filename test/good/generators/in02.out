true
0
false
true
0
false
false
